frontend/sw360-portlet/src/main/resources/META-INF/resources/html/projects/view.jsp

```diff
@@ -207,6 +222,112 @@
+                        <span aria-hidden="true">&times;</span>
+                    </button>
+                </div>
+                <div class="modal-body">
+                    <p>Fill the form to create clearing request for the project <b data-name="name"></b>?</p>
+                     <div data-hide="hasNoDependencies">
+                        <p>
+                        This project contains:
+                        </p>
+                        <ul>
+                            <li data-hide="hasNoLinkedProjects"><span data-name="linkedProjects"></span> linked projects</li>
+                            <li data-hide="hasNoLinkedReleases"><span data-name="linkedReleases"></span> linked releases</li>
+                            <li><span data-name="clearingStatus"></span></li>
+                        </ul>
+                    </div>
+                    <hr/>
+                    <form id="createClearingRequestForm">
+                        <div class="form-group">
+                            <label for="clearingTeamEmail" class="mandatory">Please enter the clearing team email id:</label>
+                            <input id="clearingTeamEmail" type="email" class="form-control" data-name="comment" rows="4" placeholder="Please enter the clearing team email" required/>
+                            <div class="invalid-feedback">
+                                The email is required in valid format!
+                            </div>
+                        </div>
+						<div class="form-group">
+							<label for="requestedClearingDate" class="mandatory">Requested Clearing Date:</label>
+							<input class="datepicker form-control" id="requestedClearingDate" type="text" pattern="\d{4}-\d{2}-\d{2}" placeholder="Requested Clearing Date YYYY-MM-DD" required/>
+							<div class="invalid-feedback">
+                                Should be greater than 7 days from today and in valid format!
+                            </div>
+                        </div>
+                        <div class="form-group">
+                            <label for="clearingRequestComment">Please comment your request</label>
+                            <textarea id="clearingRequestComment" class="form-control" data-name="comment" rows="4" placeholder="Comment your request..."></textarea>
+                        </div>
+                    </form>
+                </div>
+                <div class="modal-footer">
+                    <button type="button" class="btn btn-light" data-dismiss="modal">Close</button>
+                    <button type="button" id="createClearingrequestButton" class="btn btn-primary">Create Request</button>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <div id="viewClearingRequestDialog" class="modal" tabindex="-1" role="dialog">
+        <div class="modal-dialog modal-lg modal-dialog-centered modal-info" role="document">
```

Comment:
Please check if we can reuse functions from dialog.js and alter names according to the requirement.
Same for createClearingRequestDialog